packages/sit-onyx/src/components/TestInput/TestInput.stories.ts

```diff
@@ -9,7 +9,17 @@ const meta: Meta<typeof TestInput> = {
   title: "components/TestInput",
   ...defineStorybookActionsAndVModels({
     component: TestInput,
-    events: ["update:modelValue", "change"],
+    events: ["update:modelValue", "change", "validityChange"],
+    argTypes: {
+      type: {
+        options: ["email", "number", "password", "search", "tel", "text", "url"],
```

Comment:
We could define a Typescript type for this like following so we can use it as type for the prop and as array here:
```ts
export const INPUT_TYPES = ["email", "number", "password", "search", "tel", "text", "url"] as const;
export type InputType = typeof INPUT_TYPES[number];
```

Reply:
it's unfortunately forbidden by our eslint rules to export a const in script setup, so I had to create a second script above. hope it's fine like that